test/unit/xml_description_test.py

```diff
@@ -3,22 +3,42 @@
 
 from .test_helper import *
 
-from kiwi.exceptions import (
-    KiwiSchemaImportError,
-    KiwiValidationError,
-    KiwiDescriptionInvalid,
-    KiwiDataStructureError
-)
+from kiwi.exceptions import *
```

Comment:
Test is also code. :wink: Personally I don't do it anywhere and I always want to know what is imported. But your call...